executor/distsql.go

```diff
@@ -311,12 +309,16 @@ type TableReaderExecutor struct {
 	result        distsql.SelectResult
 	partialResult distsql.PartialResult
 	priority      int
-	feedback      *statistics.QueryFeedback
+	feedback      *variable.QueryFeedback
 }
 
 // Close implements the Executor Close interface.
 func (e *TableReaderExecutor) Close() error {
-	domain.GetDomain(e.ctx).StatsHandle().StoreQueryFeedback(e.feedback, e.result.ScanCount(), e.ranges, nil)
+	ranges := make([]interface{}, 0, len(e.ranges))
+	for _, r := range e.ranges {
+		ranges = append(ranges, r)
+	}
+	e.ctx.StoreQueryFeedback(e.feedback, e.result.ScanCount(), ranges)
```

Comment:
Why not assign the count and ranges to the feedback? 
Then `StoreQueryFeedBack` only takes one argument.